src/__tests__/Footer.test.js

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import Footer from "../components/Footer"
+
+describe("<Footer />", () => {
+  it("renders a copyright", () => {
+   
+    render(
+      <BrowserRouter>
+        <Footer />
+      </BrowserRouter>
+    )
+
+    screen.debug() 
+    screen.logTestingPlaygroundURL() 
```

Comment:
These tools are great tools when writing your tests but you would want to remove them before pushing to main.